src/generic/dcpsg.cpp

```diff
@@ -2264,16 +2264,16 @@ void wxPostScriptDCImpl::DoGetTextExtent(const wxString& string,
 
     /* return size values */
     if ( x )
-        *x = int(sum * fontSize) / 1000;
+        *x = (wxCoord) ceil(sum * fontSize / 1000.0f);
```

Comment:
Besides the added `ceil()` the rounding here is different but it restores the calculation to how it was before the bottom changes of [dcpsg.cpp](https://github.com/wxWidgets/wxWidgets/commit/b5a554b9a64ebbeeae7c7d68128f705e0ee3d4e2#diff-568c01f02e655c348ab79f317d781a7b48554884483b46ed7d5723294f4340d4), and doesn't convert to int early. So this should be OK again as the early conversion didn't seem intentional.

Reply:
The conversion to `int` was intentional, and I assert that it did not change the result of the calculation.

Reply:
Thanks for clarifying! It did assert here but not for what I would say are practical values of `sum`. What I was worried about when originally seeing the cast to `int` without commit context was that it is required and my change would break some fix. Because prior to b5a554b9a64ebbeeae7c7d68128f705e0ee3d4e2 (which helpfully didn't indicate it being for a render fix) the code was also similar to my changes (sans `ceil()`) it seemed safe but figured I would mention it anyway in case someone else noticed the same.